File: Bronze/Bronze_4/C_files/Code/Kuber_20833.c
#include <stdio.h>
#include <math.h>

int main()
{
    int n, i, sum = 0;

    scanf("%d", &n);

    for (i = 1; i <= n; i++)
        sum += round(pow(i, 3));

    printf("%d", sum);

    return 0;
}

File: Bronze/Bronze_4/Java_files/Code/Kuber_20833.java
import java.util.Scanner;
import java.lang.Math;

public class Kuber_20833 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt(), sum = 0;
        for (int i = 1; i <= n; i++)
            sum += Math.pow(i, 3);
        System.out.println(sum);
    }
}
